jenkins-master/jobs/scripts/workspace/requirements.txt

```diff
@@ -1,3 +1,3 @@
 boto==2.38.0
-mozdownload==1.18
```

Comment:
Not sure what you mean here? mozdownload is not needed for the treeherder virtualenv but mozinfo is. That's what I wrote below.